e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('I18456 Built-in slash commands: common', () => {
+    before(() => {
+        loginAndVisitDefaultChannel('user-1');
```

Comment:
Since this is applicable only for the test "/ autocomplete list can scroll" and we will unnecessarily be doing this step for the test "/shrug test", can you instead move these steps within the respective test itself instead of having it in the before method.

Reply:
No, it's also needed by the said test.